content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -165,6 +165,7 @@ different Kubernetes components.
 | `ProxyTerminatingEndpoints` | `false` | Alpha | 1.22 | |
 | `QOSReserved` | `false` | Alpha | 1.11 | |
 | `ReadWriteOncePod` | `false` | Alpha | 1.22 | |
+| `RecoverVolumeExpansionFailure` | `false` | Alpha | 1.23 | |
```

Comment:
In addition to this, we expect a brief introduction (possibly with a link to details) added at the end of this page.

Reply:
added more details.

Reply:
I don't see the extra details. This should be in the form of an edit to the page section headed “List of feature gates”.

Reply:
Added details below. PTAL.